tests/vmi_lifecycle_test.go

```diff
@@ -949,7 +950,11 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 			})
 
 			It("[test_id:3199]should set default cpu model when vmi doesn't have it set", func() {
+				if len(supportedCpuModels) < 1 {
+					Skip("Must have at least one supported cpuModel for this test")
+				}
 				config := originalConfig.DeepCopy()
+				defaultCPUModel := supportedCpuModels[0]
```

Comment:
What happens if we have a heterogeneous cluster? My concern is that the first model in `supportedCpuModels` will be supported by only one node. Maybe it's better to take the model that is supported by all (or most) nodes?

WDYT?

Reply:
Hey,
Thanks for reviewing.
i didn't preform any migration in these tests so that shouldn't be a problem.

Reply:
Yup, that's right. Thanks!